src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -229,22 +225,21 @@ else if( c instanceof IConfigurableObject )
 		}
 	}
 
-	@Override
-	public void clientPacketData( INetworkInfo network, AppEngPacket packet, EntityPlayer player )
+	public void clientHandling( PacketValueConfig message, MessageContext ctx )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "ctx". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)